download-data.py

```diff
@@ -346,7 +360,12 @@ def main() -> None:
         default="",
         help="The AWS profile to use for the download. Uses the current default profile if undefined.",
     )
-
+    parser.add_argument(
+        "--update-symlink",
+        type=str,
+        default="",
+        help="The release version to update the 'current' symlink to direct to. Release directory must be present. Data will not be re-downloaded if it is already present."
```

Comment:
This seemed to imply that data would be downloaded if not present.
```suggestion
        help="The release version to update the 'current' symlink to direct to. Release directory must be present. No data will be downloaded."
```